### PR TITLE
Update taimen, walleye for android 11

### PR DESCRIPTION
**Pull request by danielfullmer**
I'm now also setting `vendor-skip-files` and have added the required modules to `forced-modules`.

Also includes two small miscellaneous fixes described in #12

### CODE REVIEW

File: taimen/config.json [hunk 1]
```diff
@@ -61,7 +61,6 @@
         "system/lib/com.qualcomm.qti.qcril.qcrilhook@1.0.so",
         "system/lib/libdiag_system.so",
         "system/lib/libeaselcomm.so",
-        "system/lib/libeaselcontrol.amber.so",
```
Comment (chirayudesai) [inline taimen/config.json:64]: This is in vendor now (and thus would get included by default.

Reference: https://git.rip/dumps/google/taimen/-/blob/taimen-user-11-RP1A.200720.009-6720564-release-keys/all_files.txt

File: taimen/config.json [hunk 2]
```diff
@@ -89,21 +87,390 @@
...
       "BoardConfigVendor": [],
       "device-vendor": [
         "PRODUCT_PROPERTY_OVERRIDES += \\",
         "    ro.hardware.fingerprint=fpc \\",
+        "    ro.hardware.vulkan=adreno \\",
+        "    ro.hardware.egl=adreno \\",
         "    ro.gfx.driver.0=com.google.pixel.wahoo.gfxdrv \\",
         "    ro.oem_unlock.pst=/dev/block/platform/soc/1da4000.ufshc/by-name/misc \\",
-        "    ro.oem_unlock.pst_offset=6144"
+        "    ro.oem_unlock.pst_offset=6144\\",
```
Comment (chirayudesai) [inline taimen/config.json]: missing space, should be:
`"    ro.oem_unlock.pst_offset=6144 \\",`
for consistency

File: taimen/config.json [hunk 3]
```diff
@@ -89,21 +87,390 @@
...
+        "    ro.storage_manager.enabled=true"
```
Comment (chirayudesai) [inline taimen/config.json]: Should we add this here? It seems like it's something for a generic feature and not hardware specific.

**Review by chirayudesai — CHANGES_REQUESTED** [review]
Same comments apply to walleye too.

LGTM otherwise, haven't tested it yet though.

**Comment by chirayudesai** [comment]
Another thing is would you mind just doing a quick diff of walleye and taimen once you make those changes?

They shouldn't differ too much I feel, would be good to see. Thanks!

**Comment by chirayudesai** [comment]
Note: There is a new `system_ext`

On taimen/walleye it's just under system, like product. so `/system/system_ext` like `/system/product`

Not seeing anything that we'd care about in the naked config there.

There's some of the stuff that's usually in the full config though, like `CarrierSetup`

File: taimen/config.json (hunk 1 above)
Comment (danielfullmer) [reply taimen/config.json:64]: Yep, that's why this line is being removed from `system-other` here.

File: taimen/config.json (hunk 2 above)
Comment (danielfullmer) [reply taimen/config.json]: Fixed

File: taimen/config.json (hunk 3 above)
Comment (danielfullmer) [reply taimen/config.json]: Sure, removed. It's in the upstream `vendor.img` `build.prop` and I was ambivalent about adding it to our configs also.

**Comment by danielfullmer** [comment]
@chirayudesai Sure, here's a diff of their `config.json` files:
```diff
diff --git a/taimen/config.json b/walleye/config.json
index a65ec62..f25238c 100644
--- a/taimen/config.json
+++ b/walleye/config.json
@@ -1,30 +1,30 @@
 {
-  "device-name": "pixel2-xl",
-  "device-model": "taimen",
+  "device-name": "pixel2",
+  "device-model": "walleye",
   "device-aliases": [],
   "vendor": "google",
   "aosp-vendor-dir": "google_devices",
-  "device-family": "taimen",
+  "device-family": "muskie",
   "supported-apis": ["api-30"],
   "extra-partitions": [],
   "ota-partitions": [
     "abl",
-    "aes",
+    "apdp",
     "cmnlib",
     "cmnlib64",
     "devcfg",
     "hyp",
     "keymaster",
-    "laf",
     "modem",
+    "msadp",
     "pmic",
     "rpm",
     "tz",
     "xbl"
   ],
-  "AndroidMk": "taimen/Android.mk",
-  "BoardConfigVendorMk": "taimen/proprietary/BoardConfigVendor.mk",
-  "DeviceVendorMk": "taimen/proprietary/device-vendor.mk",
+  "AndroidMk": "walleye/Android.mk",
+  "BoardConfigVendorMk": "muskie/proprietary/BoardConfigVendor.mk",
+  "DeviceVendorMk": "muskie/proprietary/device-vendor-walleye.mk",
   "api-30": {
     "naked": {
       "overlays-dir": "",
@@ -444,14 +444,14 @@
         "lib/libwpa_client.so",
         "lib/mediacas/libclearkeycasplugin.so",
         "lib/mediadrm/libdrmclearkeyplugin.so",
-        "lib/modules/ftm4.ko",
-        "lib/modules/lge_battery.ko",
+        "lib/modules/htc_battery.ko",
         "lib/modules/modules.alias",
         "lib/modules/modules.dep",
         "lib/modules/modules.load",
         "lib/modules/modules.softdep",
-        "lib/modules/sw49408.ko",
-        "lib/modules/touch_core_base.ko",
+        "lib/modules/synaptics_dsx_core_htc.ko",
+        "lib/modules/synaptics_dsx_fw_update_htc.ko",
+        "lib/modules/synaptics_dsx_rmi_dev_htc.ko",
         "lib/modules/wlan.ko",
         "lib/soundfx/libaudiopreprocessing.so",
         "lib/soundfx/libbundlewrapper.so",
```

File: taimen/config.json (hunk 3 above)
Comment (chirayudesai) [reply taimen/config.json]: Yeah I was sticking with just having anything that's hardware-specific here, but good catch, we should set that in our device/ or vendor/ or something.

**Review by chirayudesai — APPROVED** [review]

**Comment by chirayudesai** [comment]
Thanks for the changes and the diff.

It looks sane, just the hardware differences between the devices (and nothing else)

**Pull request merged**